src/controller/chat-controller.js

```
@@ -0,0 +1,646 @@
+const { debugLog } = require('../utility');
+
+// Try to use the new SimpleInteractive class first, fall back to InteractiveChat if needed
+let SimpleInteractive;
+try {
+  SimpleInteractive = require('../lib/simple-interactive');
+} catch (error) {
+  utility.debugLog('SimpleInteractive not available, using InteractiveChat');
+}
+
+try {
+  InteractiveChat = require('../lib/interactive-chat');
```

Biome (1.9.4)

[error] Can't assign InteractiveChat because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)